app/models/user.rb

```diff
@@ -4,22 +4,14 @@ class User < ApplicationRecord
   has_many :accounts
   
   def create_accounts
-    # call api to get account info
-
-    # create an account for each of them
-    response = HTTParty.post('http://api119525live.gateway.akana.com:80/user/accounts',
-      :body => {LegalParticipantIdentifier: self.participant_id}.to_json,
-      :headers =>{'Content-Type' => 'application/json', 'Accept' => 'application/json'})
-
-    response["AccessibleAccountDetailList"].each do |account_data|
+    # create an account for each of users accounts in their information
+    account_details = get_account_details
+    # checks for valid account types (certain off limits for hackathon test data)
+    account_details.each do |account_data|
       valid_account_types = ['BCD', 'CCD', 'DDA']
```

Comment:
@stussy446 
You're setting the `valid_account_types` variable inside the loop, but you're just using the contents of this variable for comparison and not actually mutating it. I would suggest setting it outside of the loop so that you don't have to use resources to redeclare it for every iteration of your loop.